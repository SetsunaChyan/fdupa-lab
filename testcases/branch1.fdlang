function main(){
x = input();

if (x == 1) {
    x = 50;
} else {
    if (x == 2) {
        x = 40;
    } else {
        if (x == 3) {
            x = 30;
        } else {
            if (x == 4) {
                x = 20;
            } else {
                x = 10;
            }
            check_interval(x, 10, 20);
            check_interval(x, 20, 20);
        }
        check_interval(x, 10, 30);
        check_interval(x, 30, 40);
    }
}

check_interval(x, 10, 50);
}