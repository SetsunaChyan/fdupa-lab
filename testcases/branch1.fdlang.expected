Line 18: YES; x in [10, 10] U [20, 20]
Line 19:  NO; x in [10, 10] U [20, 20]
Line 21: YES; x in [10, 10] U [20, 20] U [30, 30]
Line 22:  NO; x in [10, 10] U [20, 20] U [30, 30]
Line 26: YES; x in [10, 10] U [20, 20] U [30, 30] U [40, 40] U [50, 50]
